src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,35 @@
+package lotto.view;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+
+  private static final Scanner SCANNER = new Scanner(System.in);
+
+  public static int readPurchaseAmount() {
+    System.out.println("구입금액을 입력해 주세요.");
+    return parseIntOrThrow(SCANNER.nextLine());
+  }
+
+  public static List<Integer> readWinningNumbers() {
+    System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+    String input = SCANNER.nextLine();
```

Comment:
```suggestion
  public static String readWinningNumbers() {
    System.out.println("지난 주 당첨 번호를 입력해 주세요.");
    return SCANNER.nextLine();
```
String으로 반환하고, Lotto와 같은 객체의 생성자가 String을 받아 값을 초기화하도록 구현하거나 or 정적 팩터링 메서드 or 팩터리 클래스 옵션 중 하나를 선택해 구현하도록 하면 어떨까?
이와 같이 구현할 경우 아래 코드는 UI와 의존하고 있기 때문에 단위 테스트하기도 힘들지 않을까?